backend/src/email/middlewares/email.middleware.ts

```diff
@@ -223,10 +223,17 @@ export const InitEmailMiddleware = (
     const { fromName: defaultFromName, fromAddress: defaultFromAddress } =
       parseFromAddress(config.get('mailFrom'))
 
+    // As part of a PSD directive, we have changed the defaultFromAddress to info@mail.postman.gov.sg.
+    // To prevent any breaking changes, we must now support both the new and old default address
+    const allowedDefaultAddresses = [
+      defaultFromAddress,
+      'donotreply@mail.postman.gov.sg',
+    ]
```

Comment:
Can we centralise this, so that we have a single source of truth?

But I won't block the PR cus of this, because we don't foresee further changes.

Reply:
Okie, I changed it to reuse `isDefaultFromAddress`. I'm pushing to staging now, lemme reverify it still works.

Reply:
I verified your initial commit works. I'll also check again once this is up on staging. It's a quick check for me - I use the transactional email API to send 4 emails and also create campaign in UI to send email

Reply:
Works for me now too